javacord-core/src/main/java/org/javacord/core/DiscordApiImpl.java

```diff
@@ -1094,6 +1106,11 @@ public String getPrefixedToken() {
         return accountType.getTokenPrefix() + token;
     }
 
+    @Override
+    public Set<Intent> getIntents() {
+        return intents;
```

Comment:
Oops, I missed this one in earlier reviews, sorry.
All collection returned by Javacord should be immutable. In this case, no defensive copy is necessary, as we never modify the set internally.
```suggestion
        return Collections.unmodifiableSet(intents);
```